aci/resource_aci_cloudctxprofile.go

```diff
@@ -339,7 +380,17 @@ func resourceAciCloudContextProfileUpdate(ctx context.Context, d *schema.Resourc
 		return diag.FromErr(fmt.Errorf("Invalid Configuration relation_cloud_rs_to_ctx property cannot be empty for the Cloud APIC"))
 	}
 
-	cloudCtxProfile, err := aciClient.UpdateCloudContextProfile(name, TenantDn, desc, cloudCtxProfileAttr, PrimaryCIDR, Region, vendor, cloudRsCtx)
+	accessPolicy := ""
+	if tmpVar, ok := d.GetOk("access_policy_type"); ok {
+		accessPolicy = tmpVar.(string)
+	}
+
+	cloudBrownfield := ""
+	if tmpVar, ok := d.GetOk("cloud_brownfield"); ok {
+		cloudBrownfield = tmpVar.(string)
+	}
```

Comment:
```suggestion
	accessPolicy := d.Get("access_policy_type").(string)
	cloudBrownfield := d.Get("cloud_brownfield").(string)
```
I think we can directly use "Get" instead of "GetOk".

Reply:
I suppose we only use Get here when the attributes are required

Reply:
if access_policy_type or cloud_brownfield is None, we cannot use .(string) here